packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/validations.ts

```diff
@@ -0,0 +1,57 @@
+import z from 'zod';
+
+export const VALIDATION_MESSAGES = {
+  NO_INPUT: 'No input',
+  MAX_CHAR_LIMIT_REACHED: 'Max character limit reached',
+  INVALID_VALUE: 'Invalid value',
+};
```

Comment:
I would move these to a more global area once we get more forms that use these same messages, but to reduce noise i won't over abstract for now.